tfjs-backend-webgpu/src/webgpu_program.ts

```diff
@@ -158,6 +174,8 @@ function makeShader(
   const prefixSnippets: string[] = [];
   const flatWorkgroupSize = program.workgroupSize[0] *
       program.workgroupSize[1] * program.workgroupSize[2];
+  program.outputComponent =
+      program.outputComponent ? program.outputComponent : 1;
```

Comment:
It's not good to set a class member outside. We can leave this for future refactoring.